### ISSUE TITLE
About the refine_margin

### ISSUE THREAD

**Issue by GalenOOO** [j96w/DenseFusion#49]
@j96w Thanks for your work! 
Here is my question. The condition that "best_test < opt.refine_margin(0.013)" is achieved after about 10 epochs, which means the posenet is just trained 10 times. However, I found the best_test is about 0.006 after training the refineNet for more than 400 times. So, is the refine_margin a little big? Can I set it smaller (eg, 0.008) to train the posenet more times? Or you have found that a refine_margin value smaller than 0.013 will lead to overfitting on the posenet?

**Comment by kaixin-bai** [comment]
Hi, i have also trained DenseFusion, but it just never achieve the 0.013, the loss is always around 0.03, did you use the train.py in densefusion or with some others parameter? thanks!

**Comment by joy-yjl** [comment]
> Hi, i have also trained DenseFusion, but it just never achieve the 0.013, the loss is always around 0.03, did you use the train.py in densefusion or with some others parameter? thanks!

Did you train the model on your own dataset？I meet the same question. Has your problem been solved?

**Comment by j96w** [comment]
Hi, sorry to miss this issue. Yes, too small refine_margin will lead to overfitting.
